charts/tfy-karpenter-config/values.yaml

```diff
@@ -184,8 +29,47 @@ karpenter:
     ## @param karpenter.defaultNodeTemplate.detailedMonitoring
     # Set this to true to enable EC2 detailed cloudwatch monitoring
     detailedMonitoring: false
-
-  ## Defautl GPU Node Template Config
+    ## @param karpenter.defaultNodeTemplate.extraSubnetTags Additional tags for the subnet.
+    extraSubnetTags: {}
+    ## @param karpenter.defaultNodeTemplate.extraSecurityGroupTags Additional tags for the security group.
+    extraSecurityGroupTags: {}
+
+  ## Default Node Pool Config
+  defaultNodePool:
+    ## @param karpenter.defaultNodePool.enabled Enable the default node pool
+    enabled: true
+    ## @param karpenter.defaultNodePool.weight Weight for the default node pool
+    weight: 10
+    ## @param karpenter.defaultNodePool.labels Labels for the default node pool
+    labels: {}
+    ## @param karpenter.defaultNodePool.taints Taints for the default node pool
+    taints: []
+    ## @param karpenter.defaultNodePool.disruption.consolidationPolicy Consolidation policy for disruption
+    disruption:
+      consolidationPolicy: WhenUnderutilized
+    ## @param karpenter.defaultNodePool.capacityTypes Capacity types for the default node pool
+    capacityTypes:
+      - spot
+    ## @param karpenter.defaultNodePool.zones Zones for the default node pool
+    zones: ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+    ## @param karpenter.defaultNodePool.architectures Architectures for the default node pool
+    architectures:
+      - amd64
+    ## @param karpenter.defaultNodePool.instanceFamilies.notAllowed Not allowed instance families for the default node pool
+    instanceFamilies:
+      notAllowed: ["t3", "t2", "t3a", "p2", "p3", "p4d", "p4de", "p5", "g4dn", "g5", "g4ad", "inf1", "inf2", "trn1", "trn1n"]
+    ## @param karpenter.defaultNodePool.instanceSizes.notAllowed Not allowed instance sizes for the default node pool
+    instanceSizes:
+      notAllowed: ["nano", "micro", "metal"]
+    ## @param karpenter.defaultNodePool.ec2NodeClassRefName EC2 Node Class Reference Name for the default node pool
+    ec2NodeClassRefName: default
```

Comment:
I am not sure of this, but this can be removed, if we are taking it from the node template name